### ISSUE TITLE
"I accept the terms of use" accessibility issues

### ISSUE THREAD

**Issue by BryceStevenWilley** [SuffolkLITLab/docassemble-AssemblyLine#800]
From the VT accessibility audit:

> On step 1, the text “To continue, please accept our terms of use” is tagged as a < label > but is not
associated with a form control. Although this is transparent to the user, as a best practice we
recommend removing the < label > structure here. See Labels. 4-Recommendation

and

> The “I accept the terms of use” checkbox is contained within a < fieldset > having a visually-hidden
< legend > of “Checkboxes (select at least one)”. Since there is only one checkbox, the legend is illogical
and the fieldset structure is unnecessary. The checkbox can stand on its own. We recommend removing
the fieldset/legend structure in this and any other instances where the “group” consists of only one
form control. 3-Moderate

Might be possible to fix upstream, but we are also using `checkboxes` here with one specific checkbox. Might be worth some extra validation code and a different datatype for better form / fieldset structure in DA.

**Comment by nonprofittechy** [comment]
This is one we should be able to fix by May 30th

**Comment by nonprofittechy** [comment]
A quick grep on the repositories I have checked out locally reveals dozens of matches, particularly in automated tests, to the current dictionary. Fixing it will break a lot of other people's existing automated tests.

One thought is to add a "feature flag" to use a new version of the question. we can add a log entry when people use an "old" interview that doesn't have the feature flag turned on. then we can gradually turn it off

**Comment by plocket** [comment]
Maybe I'm misunderstanding what we need, but as far as tests go, I suspect we can just make the variable a little funky. On the plus side, it will probably make translation easier. Here's an example of what I'm thinking if we use `yesno`:

```yml
objects:
  - acknowledged_information_use: DADict
---
question: Terms
fields:
  - I accept: acknowledged_information_use["I accept the terms of use."]
    datatype: yesno
validation code: |
```

**Comment by nonprofittechy** [comment]
I see, that would be just fine. I wasn't sure what kiln would expect there

**Comment by nonprofittechy** [comment]
Note we explored this more today and found that even a single checkbox (datatype yesno) has the redundant label "choices", which is a bit better but still redundant.

**Comment by BryceStevenWilley** [comment]
This was changed in https://github.com/SuffolkLITLab/docassemble-AssemblyLine/commit/d68351096e801292ac1fc9bcb14c30b857dd8710, however, it starts using the `validation code` feature, which doesn't announce any issue with the page when added to a field.

Definitely a big issue that should be fixed upstream (I'm not sure there's a better option without an upstream fix).
